client/app/lib/util/cleanup.coffee

```diff
@@ -38,3 +77,9 @@ module.exports = ->
     for option in options
       if kookies.expire key, option
         console.log "#{key} deleted"
+
+  for exp in regexWith
```

Comment:
you can do this with only one regex:

```regex
^(?:_hp2_(?:id|ses_props)|_ob_pub)|.*_mixpanel$
```

![regex](http://s.sinanyasar.com/170111_122748.png)

if you think this is more readable, that's fine. But either way you need to change `_mixpanel$` to `.*_mixpanel$`